helm/afm/files/conf.yaml

```diff
@@ -32,10 +32,10 @@ data:
     {{- $redactColumns := list -}}
     {{- $removeColumns := list -}}
     {{- range .transformations -}}
-      {{- if eq .id "redact-ID" -}}
+      {{- if eq .id "RedactAction" -}}
         {{- $redactColumns = append  $redactColumns .args.column_name -}}
```

Comment:
@cdoron Sorry merged by accident. 
1. `.id`: I think it should be `.action.name`
2. `.args.column_name`; I think it should be `.action.RedactAction.columns` 

@ritwikc can verify

Reply:
Yes. That's the change.